dvc/progress.py

```diff
@@ -30,12 +30,12 @@ def is_finished(self):
         """Returns if all targets have finished."""
         return self._n_total == self._n_finished
 
-    def _clearln(self):
-        self._print("\r\x1b[K", end="")
+    def clearln(self):
```

Comment:
I don't see `clearln` or `print` being used outside of the progress, why remove the leading underscore?

Reply:
@progress_aware is outside of Progress class

Reply:
@pared Got it. Thanks! :)